BraveWallet/Crypto/Stores/WalletStore.swift

```diff
@@ -5,6 +5,12 @@
 
 import Foundation
 import BraveCore
+import SwiftUI
+
+/// Sub stores under `WalletStore`
+public protocol SubStore {
+  func resetStore()
+}
```

Comment:
Hmm, why do we need a special protocol for this, why can't we just re-init all the stores using the controllers they want? All we want to do is the store to toss any local state away